src/main/kotlin/no/nav/familie/pdf/pdf/språkKonfigurasjon/SpråkKontekst.kt

```diff
@@ -0,0 +1,15 @@
+package no.nav.familie.pdf.pdf.språkKonfigurasjon
+
+object SpråkKontekst {
+    private val aktivtSpråk = ThreadLocal.withInitial { "nb" }
+
+    fun brukSpråk(): String = aktivtSpråk.get()
+
+    fun settSpråk(språk: String) {
```

Comment:
Kan følge samme standard som du har gjort over med one-line? Tenker det samme om tilbakestillSpråk også 💯 Er ikke så farlig hva du går for, men så lenge det er likt for alle metodene er jeg fornøyd!
```suggestion
    fun settSpråk(språk: String)  = this.aktivtSpråk.set(språk)
```